Kitodo/src/main/java/org/kitodo/production/forms/MigrationForm.java

```diff
@@ -413,4 +416,21 @@ public Workflow getWorkflowToUse() {
     public void setWorkflowToUse(Workflow workflowToUse) {
         this.workflowToUse = workflowToUse;
     }
+
+    private boolean isTitleValid(Template template) {
```

Comment:
Could this method be moved to `MigrationService` to keep `MigrationForm` free of program logic?